### PR TITLE
Create page for training bazaar

### PR DESCRIPTION
**Pull request by c-martinez**
TODOs:
 - [x] Include slides
 - [x] Add summary
 - [x] Link from main page

### CODE REVIEW

File: content/bazaar/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ - Exchange ideas
+
+### Programme
+
+| Presenter | Title | Slides |
+|-----------|-------|--------|
+| Fredo Erxleben (Helmholtz Federated IT Services, HIFIS), Stephanie Schworm (Helmholtz Information and Data Science Academy HIDA), Donatella Cea (Helmholtz AI) | Education and Training in HIDA, HIFIS and Helmholtz AI: Status Quo and Future Perspectives | LINK TO SLIDES |
+| Peter Pelz (NFDI Section Training & Education) | Germany’s Research Data Management Infrastructure (NFDI) and its national approach to connect training offers, teaching resources and material | LINK TO SLIDES |
+| Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | LINK TO SLIDES |
+| Radovan Bast (CodeRefinery) | CodeRefinery  | LINK TO SLIDES |
```
Comment (c-martinez) [inline content/bazaar/_index.md]:
```suggestion
| Matias Jääskeläinen (CodeRefinery) | CodeRefinery  | LINK TO SLIDES |
```
I guess it is better to add Matias as he was the speaker?

File: content/bazaar/_index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| Michel Hernandez Villanueva (DESY and HSF) | Training Activities and Experiences in the HSF | LINK TO SLIDES |
+| Danielle Sieh (The Carpentries) | The Carpentries - plans and implementation for signing-up for workshops with AMY, post teaching opportunities, match skills, interest, and availability | LINK TO SLIDES |
+
+
+### Round table discussion
+
+SUMMARY OF DISCUSSION ?
```
Comment (c-martinez) [inline content/bazaar/_index.md]:
```suggestion
### Summary of discussion
During the session it was clear that all participants share a common issue: training opportunities for RSE skills are in high demand and there is a big need for more trainings and more instructors. It was clear that there is a need for more coordination between organizations that provide trainings across Europe. Reusing training materials, automating workshop organisation, sharing experiences, are just some ideas on how this cooperation could benefit all parties involved.


You can also read the [HIFIS at the EuSSI training bazar](https://hifis.net/news/2022/11/30/HIFIS-at-EuSSI-training-bazar.html) blog post!
```

File: content/bazaar/_index.md (hunk 1 above)
Comment (c-martinez) [reply content/bazaar/_index.md]:
```suggestion
| Matias Jääskeläinen (CodeRefinery) | CodeRefinery  | [SLIDES](https://doi.org/10.5446/60140) |
```

File: content/bazaar/_index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+### Programme
+
+| Presenter | Title | Slides |
+|-----------|-------|--------|
+| Fredo Erxleben (Helmholtz Federated IT Services, HIFIS), Stephanie Schworm (Helmholtz Information and Data Science Academy HIDA), Donatella Cea (Helmholtz AI) | Education and Training in HIDA, HIFIS and Helmholtz AI: Status Quo and Future Perspectives | LINK TO SLIDES |
+| Peter Pelz (NFDI Section Training & Education) | Germany’s Research Data Management Infrastructure (NFDI) and its national approach to connect training offers, teaching resources and material | LINK TO SLIDES |
+| Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | LINK TO SLIDES |
+| Radovan Bast (CodeRefinery) | CodeRefinery  | LINK TO SLIDES |
+| Michel Hernandez Villanueva (DESY and HSF) | Training Activities and Experiences in the HSF | LINK TO SLIDES |
```
Comment (c-martinez) [inline content/bazaar/_index.md]:
```suggestion
| Michel Hernandez Villanueva (DESY and HSF) | Training Activities and Experiences in the HSF | [SLIDES](https://doi.org/10.5446/60138) |
```

File: content/bazaar/_index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ - Present training material and activities to other organizations
+ - Exchange ideas
+
+### Programme
+
+| Presenter | Title | Slides |
+|-----------|-------|--------|
+| Fredo Erxleben (Helmholtz Federated IT Services, HIFIS), Stephanie Schworm (Helmholtz Information and Data Science Academy HIDA), Donatella Cea (Helmholtz AI) | Education and Training in HIDA, HIFIS and Helmholtz AI: Status Quo and Future Perspectives | LINK TO SLIDES |
+| Peter Pelz (NFDI Section Training & Education) | Germany’s Research Data Management Infrastructure (NFDI) and its national approach to connect training offers, teaching resources and material | LINK TO SLIDES |
+| Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | LINK TO SLIDES |
```
Comment (c-martinez) [inline content/bazaar/_index.md]:
```suggestion
| Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | [SLIDES](https://doi.org/10.5446/60137) |
```

**Review by c-martinez — COMMENTED** [review]
Adding DOIs for first videos

**Comment by bast** [comment]
Thanks a lot Carlos!

**Pull request merged**